package.json

```diff
@@ -197,8 +197,9 @@
     "doctrine": "3.0.0",
     "dotparser": "1.1.1",
     "ejs": "3.1.10",
-    "embla-carousel": "8.1.8",
-    "embla-carousel-autoplay": "8.1.8",
+    "embla-carousel": "^8.2.0",
```

Comment:
I know this is confusing :D but we use fixed versions for root package.json ( in order to get a match what we actually use ), any package being shipped needs to use `^` or `~` though.

with this in place syncpack will complain -> that's why you need to add those packages to ignore ( which you already did)

Reply:
also ci is failing on invalid `yarn.lock`

```
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`
```

make sure to run `yarn` and `yarn run dedupe`